lsstsal/scripts/code/templates/Makefile.sacpp_SAL_pybind11.template1

```diff
@@ -0,0 +1,83 @@
+#----------------------------------------------------------------------------
+#       Macros
+#----------------------------------------------------------------------------
+PYTHONVER     = $(PYTHON_BUILD_VERSION)
+CC            = gcc
+CXX           = g++
+CPPFLAGS      = $(PICFLAGS) $(GENFLAGS) $(SAL_CPPFLAGS) -Wno-write-strings -std=c++11 -D_REENTRANT -Wall -I"." -I"$(OSPL_HOME)/examples/include" -I"$(OSPL_HOME)/examples" -I"$(OSPL_HOME)/include" -I"$(OSPL_HOME)/include/sys" -I"$(OSPL_HOME)/include/dcps/C++/SACPP" -I"$(PYTHON_BUILD_LOCATION)/include/python$(PYTHONVER)" -I"$(SAL_HOME)/include" -I"$(PYTHON_BUILD_LOCATION)/include/python$(PYTHONVER)" -I../../../SALSubsys/cpp/src -I../../../SALSubsys/cpp -DSAL_BUILD_FOR_PYTHON #-DSAL_SUBSYSTEM_ID_IS_KEYED
```

Comment:
There is a lot of repetition between the two template files. Would it be practical to simply define PYTHON_INCLUDE and/or PYTHON_LIB in this file if it not defined and PYTHON_BUILD_LOCATION and PYTHONVER are defined?

Reply:
You mean more like the latest commit

Reply:
Yes!

Reply:
Reverted to simpler version setting it up in the setup.env rather than fighting make to do what I want